spec/tree_spec.rb

```diff
@@ -2,13 +2,73 @@
 require 'tree'
 
 describe 'Tree' do
-  it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+	let(:tree) { Tree.new }
+  it 'is an AppleTree' do
+    expect(tree.is_a? AppleTree).to be true
+  end
+
+  it 'is alive' do
+  	expect(tree.dead?).to be false
+  end
+
+  it 'ages' do
+  	tree.age!
+  	expect(tree.age).to eq 1
+  end
+
+  it 'grows' do
+  	height = tree.height
+  	tree.age!
+  	expect(tree.height).to be > height
```

Comment:
I have no idea why its showing like that in github. In sublime it's all 2 space.